edudoc/guidelines-for-disciplinary-incidents/guidelines-for-disciplinary-incidents.md

```diff
@@ -1,93 +1,91 @@
 # Guidelines for Disciplinary Incidents
 
-The WCA Disciplinary Committee (WDC) is entitled to apply bans and other disciplinary action as a result of violation of the [WCA Code of Conduct](wcadoc{documents/Code%20of%20Conduct.pdf}), based on the motion [Suspensions and other Sanctions](wcadoc{documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf}). However, the Committee is not able to handle difficult situations quickly during a competition, especially if communication with multiple involved parties is required. Cooperation with competition Delegates is often necessary. This document contains a collection of guidelines to instruct new Delegates on how to handle common disciplinary incidents, what disciplinary measures they can take, and what they should report to the WDC.
+The WCA Integrity Committee (WIC) is entitled to apply bans and other disciplinary action as a result of violation of the [WCA Code of Conduct](wcadoc{documents/Code%20of%20Conduct.pdf}), based on the motion [Suspensions and other Sanctions](wcadoc{documents/motions/15.2022.1%20-%20Suspensions%20and%20other%20Sanctions.pdf}). However, the WIC is not able to handle difficult situations quickly during a competition, especially if communication with multiple involved parties is required. Cooperation with competition Delegates is often necessary. This document contains a collection of guidelines to instruct new Delegates on how to handle common disciplinary incidents, what disciplinary measures they can take, and what they should report to the WIC.
 
 ## Disciplinary Log and Banned Competitors
 
-* A list of banned competitors can be found in the [Disciplinary log](wca{disciplinary}). If a competitor attempts to attend a competition when they are banned from doing so, please contact the WDC.
-* Depending on the degree of severity and type of violation of the Code of Conduct, the WDC may ban a competitor from competing or attending competitions. Such a ban will be communicated with local Delegates when it is issued. If a competitor is banned from competing, but not from attending competitions, they are permitted to attend competitions. They may also fulfill the roles of officials at competitions at the discretion of the WCA Delegate.
-* The WDC will publish reports on remarkable cases they've managed and issue a quarterly digest showcasing smaller resolved cases from the previous quarter. These publications aim to educate the community and WCA staff, promoting transparency. You can find them in the announcement section on the WCA homepage and forum, with a comprehensive list available in the [Disciplinary log](wca{disciplinary}).
+- A list of banned competitors can be found in the [Disciplinary log](wca{disciplinary}). If a competitor attempts to attend a competition when they are banned from doing so, please contact the WIC.
```

Comment:
A list of banned competitors can now be found in [Delegate Panel](https://worldcubeassociation.org/panel/delegate#banned-competitors) instead of the disciplinary log.